app/views/userGuide/usingMatchers.scala.html

```diff
@@ -204,6 +204,23 @@
 the equality constraints provided by <code>TypeCheckedTripleEquals</code>, however, the statement fails to compile. For more information
 and examples, see the main documentation for <a href="@latestScaladoc/#org.scalactic.TypeCheckedTripleEquals">trait <code>TypeCheckedTripleEquals</code></a>.</p>
 
+<p>An example of TypeCheckedTripleEquals mixed in a test class:</p>
+
+<pre class="stHighlighted">
+<span class="stReserved">import</span> org.scalatest._
+<span class="stReserved">import</span> org.scalactic.TypeCheckedTripleEquals
+
+<span class="stReserved">class</span> MyTest <span class="stReserved">extends</span> FunSuite <span class="stReserved">with</span> Matchers <span class="stReserved">with</span> TypeCheckedTripleEquals {
+
+  test("Type checked ===") {
+    val i : Int    = 42
```

Comment:
Small detail, but the code convention used in the rest of the file for val declarations would be to remove the space before the `:` / after the variable name, so `val i: Int = 42` maybe?